### DISCUSSION TITLE
GitHub Education Benefits Not Applied to My Account

### DISCUSSION THREAD

**Discussion by kajun1337** [community/community#138567] [GitHub Education]
### Hi everyone,

I recently applied for the GitHub Education package and my account has been approved. However, despite receiving the approval email, I noticed that the benefits included in the package have not been applied to my account yet. I am unable to access the tools and services that are supposed to be available as part of the GitHub Education package.

Could you please assist me with this issue? If any further information is needed, I would be happy to provide it.

Thank you for your help.

**Comment by davevad93** [comment]
Hi @kajun1337 ,

Don't worry, you don't have to do anything, just wait. The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @kajun1337 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
